datasets/Van_de_Schoot_PTSD/convert_clean_schoot_lgmm_ptsd.ipynb

```diff
@@ -0,0 +1,282 @@
+      "\n",
+      "Number of papers with missing title AND abstract:  62 \n",
+      "\n",
+      "Number of papers with missing title OR abstract:   766 \n",
+      "\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# train data\n",
+    "label_inc = all_title.str.lower().isin(inc_title.str.lower())\n",
+    "label_aas = all_title.str.lower().isin(aas_title.str.lower())\n",
+    "label_dir = all_title.str.lower().isin(dir_title.str.lower())\n",
+    "\n",
+    "CODE_EXCLUDE = 0      # Exclude paper.\n",
+    "CODE_AFT_EXCLUDE = 1  # Exclude after reading full text.\n",
+    "CODE_AFT_INCLUDE = 2  # Include after reading full text.\n",
+    "CODE_AAS_INCLUDE = 3  # Include after reading abstract.\n",
+    "\n",
+    "inclusion_code = (label_aas & ~label_inc)*CODE_AFT_EXCLUDE\n",
+    "inclusion_code = inclusion_code + (label_inc & ~label_dir)*CODE_AFT_INCLUDE\n",
+    "inclusion_code = inclusion_code + label_dir*CODE_AAS_INCLUDE\n",
+    "\n",
+    "train_data = all_data.assign(included = label_inc.astype(int), inclusion_code = inclusion_code)\n",
+    "train_data = train_data.assign(authors = train_data['authors'].str.replace(\"[\\\\[']\", \"\", regex=True))\n",
+    "train_data = train_data.assign(authors = train_data['authors'].str.replace(\"\\\\]\", \"\", regex=True))\n",
+    "\n",
+    "# 64 with missing title\n",
+    "print(\"Number of papers with missing title:              \", all_title.isna().sum(), \"\\n\")\n",
+    "\n",
+    "# 762 with missing abstract\n",
+    "print(\"Number of papers with missing abstract:           \", all_data['abstract'].isna().sum(), \"\\n\")\n",
+    "\n",
+    "# 62 with both missing titles and abstracts\n",
+    "print(\"Number of papers with missing title AND abstract: \", (all_title.isna() & all_data['abstract'].isna()).sum(), \"\\n\")\n",
+    "\n",
+    "# 764 with either missing titles or abstracts\n",
+    "print(\"Number of papers with missing title OR abstract:  \", (all_title.isna() | all_data['abstract'].isna()).sum(), \"\\n\\n\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Total remaining papers in training set:      5782 \n",
+      "\n",
+      "Total number of INCLUSIONS:                  38  ( 0.66 % )\n",
+      "\n",
+      "Total number of EXCLUSIONS:                  5744 \n",
+      "\n",
+      "\n",
+      "Total EXCLUSIONS after abstract screening:   5426 \n",
+      "\n",
+      "Total EXCLUSIONS after full text screening:  318 \n",
+      "\n",
+      "Total INCLUSIONS after full text screening:  30 \n",
+      "\n",
+      "Total INCLUSIONS after abstract screening:   8 \n",
+      "\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# remove duplicates based on just titles\n",
+    "unique_train_data = train_data[(~all_title.str.lower().duplicated()) | all_title.str.lower().isnull()]\n",
```

Comment:
I'm not sure but I'm wondering: wouldn't removing entries with duplicate titles also remove entries with missing titles? Also, it would be nice to have some statistics here on how many duplicates there actually are, just as with the missingness above.

Reply:
> wouldn't removing entries with duplicate titles also remove entries with missing titles? 

Yes, but we don't want to remove entries with missing titles, right?

Reply:
Correct, I was misunderstanding what the line of code did, excuse me!